continuation.jl

```diff
@@ -1,14 +1,40 @@
 using PseudoArcLengthContinuation, Flux
-using PseudoArcLengthContinuation: ContResult,AbstractLinearSolver,EigenSolver
+using PseudoArcLengthContinuation: ContResult, DefaultLS, DefaultEig
 const Cont = PseudoArcLengthContinuation
 
+import Base: copyto!
+copyto!(x::TrackedArray, y::TrackedArray) where T = x = copy(y)
+
+import PseudoArcLengthContinuation: Newton, minus!
+minus!(x::TrackedArray, y::TrackedArray) = x = x .- y
+minus!(x::TrackedArray, y) = x = x .- y
```

Comment:
@rveltz this is not required anymore as your code works swimmingly with `Array{TrackedReal}` types

Reply:
I am surprised that this is true. On my computer, it seems I still the following:

```
import PseudoArcLengthContinuation: minus!
minus!(x::TrackedArray, y) = x = x .- y
```

Note that something is "wrong" with your setting, you should have to define `minus!(x::TrackedArray, y::TrackedArray)`. But if you do so, the function `newton` does not work. This is because:

```
# jacobian matrix
J = fdTracked((u) -> rates(u,p₀,θ...), u₀)
# linear solve used in newton function
sol = J \ rates(u₀,p₀,θ...)
typeof(u₀) == typeof(sol) #return false
```